libs/estdlib/src/erl_features.erl

```diff
@@ -0,0 +1,25 @@
+%
+% This file is part of AtomVM.
+%
+% Copyright 2024 Jakub Gonet <jakub.gonet@swmansion.com>
+%
+% Licensed under the Apache License, Version 2.0 (the "License");
+% you may not use this file except in compliance with the License.
+% You may obtain a copy of the License at
+%
+%    http://www.apache.org/licenses/LICENSE-2.0
+%
+% Unless required by applicable law or agreed to in writing, software
+% distributed under the License is distributed on an "AS IS" BASIS,
+% WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+% See the License for the specific language governing permissions and
+% limitations under the License.
+%
+% SPDX-License-Identifier: Apache-2.0 OR LGPL-2.1-or-later
+%
+
+-module(erl_features).
+
+-export([keywords/0]).
+
+keywords() -> ['maybe', 'else'].
```

Comment:
This probably needs a config option based on different OTP versions. Not sure what's the policy of supported OTP versions here.

Reply:
I needed more stubs for Cardputer example code: all/0 and configurable/0

https://github.com/pguyot/cardputer_erlang/blob/main/src/erl_features.erl